src/pages/experiments/selected/details/index.js

```diff
@@ -31,7 +31,7 @@ let Details = ({ experiment }) => (
         <ExperimentLink experiment={experiment} />
       </span>
       <span
-        aria-label={'Showing ' + (experiment.samples?.length || 0) + ' samples'}
+        aria-label={(experiment.samples?.length || 0) + ' samples'}
```

Comment:
Minor issue: When the experiment has only one sample, `1 samples` is a grammar error.